styles.css

```diff
@@ -51,8 +56,13 @@ body {
   height: 150px;
   width: 150px;
   padding: 20px;
+  display: flex;
+  flex-direction: column;
 }
 
+.child-3 {
+  margin-top: auto;
```

Comment:
Instead of using a margin, you could use the flex-grow property like this:
```
.child-2 {
  flex-grow: 1;
}
```

This way, the child-2 will take all the available space.